zk_toolbox/crates/zk_inception/src/utils/consensus.rs

```diff
@@ -41,10 +41,12 @@ pub fn get_consensus_config(
 
     let public_addr = SocketAddr::new(CONSENSUS_PUBLIC_ADDRESS_HOST, ports.consensus_port);
     let server_addr = SocketAddr::new(CONSENSUS_SERVER_ADDRESS_HOST, ports.consensus_port);
+    let debug_page_addr = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)), 5000);
```

Comment:
You shouldn't set a fixed port for the debug page. Please consult @Deniallugo on how to set it up correctly.

Reply:
exactly, it's better to introduce this port to the configs, otherwise, we will meet the port overlap

Reply:
To [here](https://github.com/matter-labs/zksync-era/blob/4ffbf426de166c11aaf5d7b5ed7d199644fba229/zk_toolbox/crates/config/src/general.rs#L207)?

Reply:
I'd rather at it at least to the consensus config files

Reply:
ConsensusConfig is what is output from this function.

Reply:
oh, ok than yes

Reply:
I ended up making the debug page optional in the code, and it's set to None for zk_toolbox. I need to take a look at zk_toolbox anyway, so I'll handle this port issue on that future PR.
For now, it will only be possible to expose the debug page on ENs started from Docker.